File: server/order-service/src/main/java/com/sudo248/invoiceservice/controller/dto/SupplierProductDto.java
package com.sudo248.invoiceservice.controller.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class SupplierProductDto implements Serializable {
    private SupplierDto supplier;
    private ProductDto product;
    private RouteDto route = new RouteDto();
    private int amountLeft;
    private double price, soldAmount, rate;
}

File: server/cart-service/src/main/kotlin/com/sudoo/cartservice/controller/dto/SupplierProductDto.kt
package com.sudoo.cartservice.controller.dto


data class SupplierProductDto (
    val supplier: SupplierDto,
    val product: ProductDto,
    val route:RouteDto = RouteDto(),
    val amountLeft:Int = 0,
    val price:Double = 0.0,
    val soldAmount:Double = 0.0,
    val rate:Double = 0.0
)
